lte/gateway/c/oai/tasks/s6a/CMakeLists.txt

```diff
@@ -22,7 +21,8 @@ if (S6A_OVER_GRPC)
   set (S6A_SRC ${S6A_SRC} s6a_grpc_iface.cpp)
   add_library(TASK_S6A ${S6A_SRC})
   target_link_libraries(TASK_S6A
-    COMMON
+  TASK_MME_APP
```

Comment:
Why do we need TASK_MME_APP here?

Reply:
I added mme_app_ue_conetxt.h in s6a_auth_info.c which had dependency on  mme_app_sgs_fsm.h so added this task here.

Reply:
Removed